2020-10-5 Build Failed plugin-config.yml missing "buildNumber" and contains "version"

** Expected Behavior**
Successful Build

** Actual Behavior**
Error: Additional properties are not allowed ('version' was unexpected) on ['additionalProperties']
Error: 'buildNumber' is a required property on ['required']

Validation failed on /Users/ranzo.taylor/Projects/couchbase-plugin/plugin_config.yml. 
0 Warning(s). 2 Error(s) 

BUILD FAILED.


**Steps To Reproduce the Problem**
git clone 
dvp build 

**Version**
dvp, version 2.1.0
couchbase-plugin commit cc40af21783eabf7c7722067fc806f187f5d936a

--- a/plugin_config.yml
+++ b/plugin_config.yml
@@ -1,6 +1,7 @@
 id: 18f4ff11-b758-4bf2-9a37-719a22f5a4b8
 name: couchbase
-version: 1.0.0
+externalVersion: 1.0.0
+buildNumber: 1.0.0
 language: PYTHON27
 hostTypes:
 - UNIX

--- a/test/requirements.txt
+++ b/test/requirements.txt
@@ -1,4 +1,4 @@
-dvp==1.0.0
+dvp==2.1.0
 pytest==4.6.10
 pytest-cov
 pytest-html

--- a/test/virtualEnvSetup.sh
+++ b/test/virtualEnvSetup.sh
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 #this script creates a virtual environment and install the below packages:
-#dvp==1.0.0
+#dvp==2.1.0
 #pytest==4.6.10
 #pytest-cov
 #pytest-html

